igvc_platform/src/imu/SerialInterface.h

```diff
@@ -13,9 +13,9 @@ class SerialInterface
   // baudrate
   int baud_;
   // connection related variables
-  SerialPtr connection_port;
+  SerialPtr connection_port_;
   std::string port_;
-  bool connected = false;
+  bool connected_ = false;
   // logger zone
   const std::string log_zone_;
```

Comment:
What is this `log_zone_` for?

Reply:
Its actually a nice idea. It's placed in front of the `ROS_INFO_STREAM` messages so that it is easy to identify where the error or message is coming from. Honestly this was a standard we implemented.